in_text/text_04_predictions.yml

```diff
@@ -315,7 +315,7 @@ entities:
       data-max: NA
       data-units: NA
       
-build-environment: Multiple computer systems were used to generate these data, including Windows, OSX, and linux. The open source languages R and Python was used on all systems.
+build-environment: Multiple computer systems were used to generate these data, including Windows, OSX, and Linux. The open source languages R and Python was used on all systems.
```

Comment:
I don't understand the pattern of when the OS info is about the data release vs. when it is about how the data itself was created